### PR TITLE
Cortx 30403 code refactor/update fix codacy issues

### PR DESCRIPTION
**Pull request by DhobaleVishal**
# Problem Statement
-  Refactored corio code.
- Fixed codacy issues.
- Improvements in reporting.
- Fixed collect_resource_utilisation.
- Improved root logging from utils.

# Design
-  For Bug, Describe the fix here.
-  For new test addition post the link for design

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed
-  [x] Attach test execution logs
https://jts.seagate.com/secure/attachment/513122/PR38_execution_logs.zip
# Review Checklist 
  Checklist for Author
-  [x] JIRA number/GitHub Issue added to PR
-  [x] PR is self reviewed
-  [x] Jira and state/status is updated and JIRA is updated with PR link
-  [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
-  [x] Changes done to ReadMe

### CODE REVIEW

File: src/commons/yaml_parser.py [hunk 1]
```diff
@@ -51,28 +52,28 @@ def convert_to_bytes(size):
     can be provided as byte(s), kb, kib, mb, mib, gb, gib, tb, tib
     :return equivalent bytes value for object size.
     """
-    kb = 1000
-    kib = 1024
```
Comment (ketanarlulkar) [inline src/commons/yaml_parser.py:55]: Instead of removing those, we can move this to constants file, since these values are also used in unit tests.

Comment (DhobaleVishal) [reply src/commons/yaml_parser.py:55]: done

File: src/commons/yaml_parser.py [hunk 2]
```diff
@@ -51,28 +52,28 @@ def convert_to_bytes(size):
...
+        size_bytes = int(size.split('kib')[0]) * 1024
     if 'mb' in size:
-        return int(size.split('mb')[0]) * kb * kb
+        size_bytes = int(size.split('mb')[0]) * 1000 * 1000
     if 'mib' in size:
-        return int(size.split('mib')[0]) * kib * kib
+        size_bytes = int(size.split('mib')[0]) * 1024 * 1024
     if 'gb' in size:
-        return int(size.split('gb')[0]) * kb * kb * kb
+        size_bytes = int(size.split('gb')[0]) * 1000 * 1000 * 1000
```
Comment (ketanarlulkar) [inline src/commons/yaml_parser.py]: Use the constant.

File: src/commons/utils/corio_utils.py [hunk 3]
```diff
@@ -134,3 +131,30 @@ def create_file(path: str, size: int):
...
+        size_name_1024 = ("B", "KiB", "MiB", "GiB", "TiB", "PiB")
+        size_name_1000 = ("B", "KB", "MB", "GB", "TB", "PB")
+        if (size_bytes % 1000) == 0:
+            check_pow = int(math.floor(math.log(size_bytes, 1000)))
+            power = math.pow(1000, check_pow)
+            size = int(round(size_bytes / power, 2))
+            part_size = f"{size}{size_name_1000[check_pow]}"
+        elif (size_bytes % 1024) == 0:
+            check_pow = int(math.floor(math.log(size_bytes, 1024)))
+            power = math.pow(1024, check_pow)
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py]: Use the defined constants

File: src/commons/utils/corio_utils.py [hunk 4]
```diff
@@ -134,3 +131,30 @@ def create_file(path: str, size: int):
...
+def convert_size(size_bytes):
+    """
+    Convert byte size to KiB, MiB, KB, MB etc.
+
+    :param size_bytes: Size in bytes.
+    """
+    if size_bytes:
+        size_name_1024 = ("B", "KiB", "MiB", "GiB", "TiB", "PiB")
+        size_name_1000 = ("B", "KB", "MB", "GB", "TB", "PB")
+        if (size_bytes % 1000) == 0:
```
Comment (ketanarlulkar) [inline src/commons/utils/corio_utils.py]:
```suggestion
        if (size_bytes % Kb) == 0:
```

**Review by ketanarlulkar — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

**Pull request merged**